docs/index.md

```diff
@@ -8,10 +8,10 @@ title: Haal Centraal BRP Personen
 ![ci]({{ site.repoUrl}}/workflows/ci/badge.svg)
 
 API voor het zoeken en raadplegen van actuele personen, partners, ouders en kinderen uit de basisregistratie personen (BRP), inclusief de registratie niet-ingezeten (RNI).
-De API biedt naast persoonsgegevens uit de BRP ook de informatieproducten leeftijd, aanschrijfwijze, aanhef, een verwijzing naar een persoon in de tekst van een brief, en adressering (passend op een vensterenvelop). Zo kan RvIG beter voorzien in klantbehoeften, en hoeven onze klanten minder persoonsgegevens te verwerken. Hiervoor is een wetwijziging nodig: het [Experimentbesluit Dataminimalisatie](https://www.rijksoverheid.nl/documenten/besluiten/2023/08/28/experimentbesluit-brp-dataminimalisatie-amvb-nvt-versie-voorhang). 
+De {{ site.apiname }} biedt naast persoonsgegevens uit de BRP ook de informatieproducten gezag, leeftijd, aanschrijfwijze, aanhef, een verwijzing naar een persoon in de tekst van een brief, en adressering (passend op een vensterenvelop). Zo kan RvIG beter voorzien in klantbehoeften, en hoeven onze klanten minder persoonsgegevens te verwerken. De juridische grondslag hiervoor is het [Experimentbesluit Dataminimalisatie]([https://www.tweedekamer.nl/kamerstukken/brieven_regering/detail?id=2023Z14338&did=2023D34429](https://zoek.officielebekendmakingen.nl/stb-2024-96.html). 
 
 ## Planning en Roadmap
-De API is live sinds mei 2023. Verwacht wordt dat het Experimentbesluit Dataminimalisatie in het tweede kwartaal van 2024 in werking treedt. Tot die tijd gebruiken afnemers een "BrpProxy API" die de bewerking tot informatieproducten bij afnemers uitvoert. De "BrpProxy API" wordt door RvIG verstrekt. 
+De BRP API is live sinds mei 2023 en bestaat uit de onderdelen Personen, [Bewoning]({{ site.pagesBaseUrl }}/Haal-Centraal-BRP-bewoning){:target="_blank" rel="noopener"} en [Reisdocumenten]({{ site.pagesBaseUrl }}/Haal-Centraal-Reisdocumenten-bevragen){:target="_blank" rel="noopener"}. Rond 1 juli wordt [Verblijfplaatshistorie]({{ site.pagesBaseUrl }}/Haal-Centraal-BRP-historie-bevragen){:target="_blank" rel="noopener"} verwacht, waarmee de verblijfplaatsen van een persoon kunnen worden opgevraagd in een bepaalde periode. 
```

Comment:
Is het misschien handig om 2024 bij 1 juli toe te voegen? Ik weet niet of lezers dit als 1 juli 2023 kunnen interpreteren

Reply:
aangepast